pkg/fetch/cisa/coun7er/coun7er.go

```diff
@@ -0,0 +1,103 @@
+package coun7er
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	"github.com/MaineK00n/vuls-data-update/pkg/fetch/util"
+	utilhttp "github.com/MaineK00n/vuls-data-update/pkg/fetch/util/http"
+)
+
+const dataURL = "https://github.com/cisagov/playbook-ng/raw/refs/heads/main/shared/data/datasets/coun7er/latest.json"
+
+type options struct {
+	dataURL string
+	dir     string
+	retry   int
+}
+
+type Option interface {
+	apply(*options)
+}
+
+type dataURLOption string
+
+func (u dataURLOption) apply(opts *options) {
+	opts.dataURL = string(u)
+}
+
+func WithDataURL(url string) Option {
+	return dataURLOption(url)
+}
+
+type dirOption string
+
+func (d dirOption) apply(opts *options) {
+	opts.dir = string(d)
+}
+
+func WithDir(dir string) Option {
+	return dirOption(dir)
+}
+
+type retryOption int
+
+func (r retryOption) apply(opts *options) {
+	opts.retry = int(r)
+}
+
+func WithRetry(retry int) Option {
+	return retryOption(retry)
+}
+
+func Fetch(opts ...Option) error {
+	options := &options{
+		dataURL: dataURL,
+		dir:     filepath.Join(util.CacheDir(), "fetch", "cisa", "coun7er"),
+		retry:   3,
+	}
+
+	for _, o := range opts {
+		o.apply(options)
+	}
+
+	if err := util.RemoveAll(options.dir); err != nil {
+		return errors.Wrapf(err, "remove %s", options.dir)
+	}
+
+	log.Println("[INFO] Fetch CISA Coun7er")
```

Comment:
```suggestion
	log.Println("[INFO] Fetch CISA  COUN7ER")
```

or all-lower `coun7er`?